yarn-project/archiver/src/archiver/archiver_store.ts

```diff
@@ -20,14 +19,9 @@ import { ContractClassPublic, ContractInstanceWithAddress } from '@aztec/types/c
  */
 export type ArchiverL1SynchPoint = {
   /** The last L1 block that added a new L2 block.  */
-  addedBlock: bigint;
-  /** The last L1 block that added messages from the new inbox. */
-  // TODO(#4492): Clean this up and fix the naming
-  newMessages: bigint;
-  /** The last L1 block that added pending messages */
-  addedMessages: bigint;
-  /** The last L1 block that cancelled messages */
-  cancelledMessages: bigint;
+  blocks: bigint;
```

Comment:
Using `blocks` (multiple) here can make it read oddly as commented higher up.

Reply:
Will address in a separate PR